### ISSUE TITLE
Asian language languages support (Japanese, Chinese, Korean) on Android & iOS

### ISSUE THREAD

**Issue by support-fennecx** [support-fennecx/advancedinputfield2#7]
**Is your feature request related to a problem? Please describe.**
Check if asian language languages can be supported (Japanese, Chinese, Korean) on Android & iOS since this gets handled by the native inputfield implementation on those platforms.

**Describe the solution you'd like**
Research the encoding of those languages.
If a character sequence can represent one glyph in those languages extend the emoji support module.
Otherwise check the steps needed to properly support the language
Either way document the steps on how to support these languages

**Comment by support-fennecx** [comment]
Seems like on Android it works out of the box with Unity Text Renders.
To get this working with the "Chat" sample scene and TextMeshPro I did the following steps:
1. Download a Japanese font. I used: https://www.freejapanesefont.com/yusei-magic-download/
2. Import the font in Unity
3. Right click on the font file in the Unity Editor and select "Create" => "TextMeshPro" => "Font Asset"
4. Assigned this font on the TextRenderers of the AdvancedInputField in that scene (Text, Processed & Placeholder child objects).
5. Made a build and ran this on my Android device

I can see the japanese glyphs and emojis appearing in the text.

Could someone who can read Japanese test if this works as it should.
